### ISSUE TITLE
Mod incompatibilities

### ISSUE THREAD

**Issue by DanDragonWolf** [Hedaox/ninjinknockback#2]
Mods that add custom forms, such as Jwan's DBC Essentials, don't work correctly with this mod because most custom form mods have their own stat modifiers that aren't reflected in the player's battle power.

**Comment by Hedaox** [comment]
Yeah I think this will need a specific implementation, and I will be forced to add dependency on Jwan's mod. So this will need another build just for that. I can't see this coming soon.
